docs/changelog.md

```diff
@@ -1,4 +1,9 @@
 # TBD
+### Features
+* Add `lambda_configurator` interface; users should create custom implementation of this to configure their own Lambda
+* Add `lambda_executor` which is responsible for starts the server, using an implementation of `lambda_configurator`, with the Lambda implementation
```

Comment:
"Add `lambda_executor` which accepts a `lambda_configurator` implementation and is responsible for starting the Lambda server"